core/src/main/java/com/spotify/metrics/core/SemanticMetricBuilderFactory.java

```diff
@@ -0,0 +1,39 @@
+package com.spotify.metrics.core;
+
+import com.codahale.metrics.Histogram;
+import com.codahale.metrics.Metric;
+import com.codahale.metrics.Reservoir;
+import com.codahale.metrics.Timer;
+import java.util.function.Supplier;
+
+public class SemanticMetricBuilderFactory {
```

Comment:
Nice this is much simpler

Reply:
Agreed. Would rather have readable, duplicated code than what I had before.